File: examples/golang/datadog-integration.go
package golang

import (
	mdb "github.com/mongodb/awscdk-resources-mongodbatlas-go/awscdkresourcesmongodbatlas"
)

type CdkTestAppGoStackProps struct {
	awscdk.StackProps
}

func NewCdkTestAppGoStack(scope constructs.Construct, id string, props *CdkTestAppGoStackProps) awscdk.Stack {
	var sprops awscdk.StackProps
	if props != nil {
		sprops = props.StackProps
	}
	stack := awscdk.NewStack(scope, &id, &sprops)

	ddProps := mdb.DatadogIntegrationProps{
		ProjectId: jsii.String("6435ce7777777777731"),
		ApiKey:    jsii.String("....."),
		Region:    mdb.DatadogRegion_US,
		Profile:   jsii.String("atlas-secret-profile-name"),
	}
	mdb.NewDatadogIntegration(stack, jsii.String("DatadogIntegration-go"), &ddProps)

	return stack
}

File: examples/python/l2-resources/datadog_integration.py
from aws_cdk import Stack
from constructs import Construct

from awscdk_resources_mongodbatlas import ( DatadogIntegration, DatadogRegion )

class CdkTestAppPython2Stack(Stack):

  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
    super().__init__(scope, construct_id, **kwargs)

    project_id_var = self.node.try_get_context('project_id')
    api_key_var = self.node.try_get_context('api_key')
    profile_name_var = self.node.try_get_context('profile')

    datadog_integration_l2 = DatadogIntegration(self, "DatadogIntegration-py-l2",
                                                api_key=api_key_var,
                                                profile=profile_name_var,
                                                project_id=project_id_var,
                                                region=DatadogRegion.US)


